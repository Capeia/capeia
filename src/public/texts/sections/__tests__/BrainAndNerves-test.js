// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<BrainAndNerves>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/sections/BrainAndNerves')
    const Component = require('public/texts/sections/BrainAndNerves').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
