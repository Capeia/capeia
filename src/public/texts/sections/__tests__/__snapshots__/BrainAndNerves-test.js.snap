// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrainAndNerves> renders correctly 1`] = `
<div>
  <p>
    <strong>
      Sensory neurons in a specimen of 
      <em>
        Caenorhabditis elegans
      </em>
       are marked with green fluorescent protein.
    </strong>
  </p>
  <p>
    <em>
      C. elegans
    </em>
    , as it is known, is a small 1 mm nematode that lives in soil and is a favourite model organism of developmental biologists and neuroscientists. It is one of the simplest animals with a nervous system. In the common hermaphrodite form it boasts 302 neurons, which is nearly one third of its total cell count of 959. This may seem a far cry from a human’s estimated 85 billion neurons but those 302 nerve cells are sufficient to endow the creature with a diverse behavioural repertoire that includes oriented movement, feeding, mating, and an escape reaction from predacious fungi. Not only is the worm’s exact number of neurons known but also its precise wiring diagram, the connectome. This neuronal map, the worm’s mind, if you like, has recently been uploaded by the 
    <a
      href="http://www.openworm.org/science.html"
      target="_blank"
    >
      OpenWorm initiative
    </a>
     into a Lego robot, resulting in what one might call a 
    <em>
      C. elegans
    </em>
     in a Lego body. The ultimate objective is a complete simulation of 
    <em>
      C. elegans
    </em>
     as a virtual organism at the cellular level. Exciting times lie ahead!
  </p>
  <p>
    <em>
      (Image credit: Heiti Paves, Science Photo Library)
    </em>
  </p>
</div>
`;
